src/hooks/posters.ts

```diff
@@ -170,14 +262,22 @@ export const usePosters = (posterHallId: string) => {
     isPostersLoaded,
     hasHiddenPosters,
 
+    categoryList,
+    subCategoryList,
+
     searchInputValue,
+    categoryFilter,
+    subCategoryFilter,
     liveFilter,
     bookmarkedFilter,
 
     increaseDisplayedPosterCount,
     setSearchInputValue,
     setLiveFilter,
     setBookmarkedFilter,
+    setCategoryFilter,
+    setSubCategoryFilter,
+    unsetCategoryFilter,
```

Comment:
Subcategory is a single word, so shouldn't be cased as 'SubCategory':

- https://en.wikipedia.org/wiki/Subcategory